### PR TITLE
Tech preview designations for pipelines, disable pipeline menu item by default

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin-web-console/issues/266

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

File: app/styles/_core.less [hunk 1]
```diff
@@ -115,6 +115,7 @@
 .breadcrumb {
   font-size: @font-size-base - 1;
   padding-bottom: 0;
+  margin-bottom: 0;
```
Comment (jwforres) [inline app/styles/_core.less]: the bottom margin on breadcrumbs didn't actually have an effect with the h1 as the next element, and looks fine when the alerts show up

Comment (spadgett) [inline app/styles/_core.less]: Won't this change the margin under the breadcrumbs on non-pipeline pages?

Comment (spadgett) [inline app/styles/_core.less]: > the bottom margin on breadcrumbs didn't actually have an effect with the h1 as the next element

Does the "Loading..." message have margin?

**Comment by spadgett** [comment]
[merge]

**Pull request merged**